python/llm/example/GPU/LLM-Finetuning/README.md

```diff
@@ -8,6 +8,7 @@ This folder contains examples of running different training mode with BigDL-LLM
 - [ReLora](ReLora): examples of running ReLora finetuning
 - [DPO](DPO): examples of running DPO finetuning
 - [common](common): common templates and utility classes in finetuning examples
+- [patch usage](patch_usage): examples of how to easily run finetuning on Intel GPU with BigDL-LLM patch
```

Comment:
`[HF-PEFT](PEFT): run finetuning on Intel GPU using Hugging Face PEFT code without modification`

Reply:
> [HF-PEFT](PEFT): run finetuning on Intel GPU using Hugging Face PEFT code without modification

Have modified example folder name and comment. Besides, considering https://github.com/analytics-zoo/nano/issues/1112, we'd better not include english_quotes dataset in our examples, so I use alpaca lora example to demonstrate usage instead.